packages/loader/container-loader/src/container.ts

```diff
@@ -310,8 +311,12 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         return PerformanceEvent.timedExecAsync(container.logger, { eventName: "Load" }, async (event) => {
             return new Promise<Container>((res, rej) => {
                 const version = loadOptions.version;
+
                 // always load unpaused with pending ops
-                const pause = pendingLocalState !== undefined ? false : loadOptions.pause;
+                const defaultMode: IContainerLoadMode = { opsBeforeReturn: "cached" };
+                const mode: IContainerLoadMode = pendingLocalState !== undefined ?
+                    defaultMode :
```

Comment:
>defaultMode [](start = 20, length = 11)

seems like we should have a more robust merge logic here, why override the deltaconnection mode in this case

Reply:
This is unfinished feature (that was canned till later time). It relies on a fact that it can apply locally saved ops before container is returned back to the host (to be able to make any additional local changes). As such, it needs to process remote ops in certain range of sequence numbers right away, as part of Cotainer.load. So it can't really accept any load state other than immediate connection.
I'll change it to an assert instead.